### DISCUSSION TITLE
Several Markers with nested InfoWindow

### DISCUSSION THREAD

**Discussion by mkra** [inocan-group/vue3-google-map#136] [General]
I noticed that InfoWindows aren't closed when you open an other InfoWindow if more than one Marker with a nested InfoWindow is used.
Code Snippet:
`<MarkerCluster :options="clusterOptions">
      <Marker
        v-for="(item, index) in markers"
        :key="index"
      >
          <InfoWindow>
             Dummy Content passed through slot
          </InfoWindow>
      </Marker>
</MarkerCluster>`
![Bildschirmfoto 2023-06-02 um 10 09 25](https://github.com/inocan-group/vue3-google-map/assets/1183873/e4fbbfa4-7c3a-4555-b868-39a96538b305)

**Comment by HusamElbashir** [comment]
I would argue this is the expected behavior but happy to be proven otherwise

**Comment by mkra** [comment]
If you click on markers on a normal google map (maps.google.com), the InfoWindow closes when you click an other marker. Or when in the example in the documentation (https://vue3-google-map.netlify.app/components/info-window.html#use-with-marker) if you zoom in and click the red marker and then a blue one or so from google, the InfoWindow closes. This behaviour is different if you add more markers.

**Reply by vdxas** [reply]
Did you found a solution?
